stubs/opentracing/opentracing/mocktracer/tracer.pyi

```diff
@@ -1,10 +1,26 @@
+from typing import Any
+
 from ..scope_manager import ScopeManager
 from ..span import Span
-from ..tracer import Tracer
+from ..tracer import Reference, Tracer
+from .context import SpanContext
 from .propagator import Propagator
+from .span import MockSpan
 
 class MockTracer(Tracer):
     def __init__(self, scope_manager: ScopeManager | None = ...) -> None: ...
+    @property
+    def active_span(self) -> MockSpan | None: ...
     def register_propagator(self, format: str, propagator: Propagator) -> None: ...
-    def finished_spans(self) -> list[Span]: ...
+    def finished_spans(self) -> list[MockSpan]: ...
     def reset(self) -> None: ...
+    def start_span(
```

Comment:
If `child_of` really needs to be a mock `SpanContext` and not `opentracing.span.SpanContext`, you need to ignore the mypy warning:

```suggestion
    def start_span(  # type: ignore[override]
```